### PR TITLE
Update System.Drawing.Common README.md

### PR DESCRIPTION
**Pull request by KlausLoeffelmann**
This fixes https://github.com/dotnet/winforms/issues/9729.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/9832)

### CODE REVIEW

File: src/System.Drawing.Common/README.md [hunk 1]
```diff
@@ -12,5 +81,10 @@ Documentation can be found at https://learn.microsoft.com/en-us/dotnet/api/syste
 
 See the [Help Wanted](https://github.com/dotnet/runtime/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-System.Drawing+label%3A%22help+wanted%22) issues.
 
-## Deployment
-`System.Drawing.Common` is provided as a [NuGet package](https://www.nuget.org/packages/System.Drawing.Common) and part of the `Microsoft.WindowsDesktop.App` shared framework.
+## Feedback
+
+For feedback, you can:
+
+- Open an issue on the [GitHub repository](https://github.com/dotnet/runtime/issues)
```
Comment (lonitra) [inline src/System.Drawing.Common/README.md:88]: Should this be a link to our repo now?

Comment (KlausLoeffelmann) [reply src/System.Drawing.Common/README.md:88]: Ups. Copy/Paste bug... 😸

File: src/System.Drawing.Common/README.md [hunk 2]
```diff
@@ -1,9 +1,78 @@
...
+    End Sub
+End Module
+```
+
+## Additional Documentation
+
+For more in-depth tutorials and API references, you can check the following resources:
+
+- [NuGet Gallery | System.Drawing.Common](https://nuget.org/packages/System.Drawing.Common/)
+- [System.Drawing.Common Namespace | Microsoft Docs](https://docs.microsoft.com/en-us/dotnet/api/system.drawing?view=net-5.0)
```
Comment (Tanya-Solyanik) [inline src/System.Drawing.Common/README.md:74]: Please remove `en-us` from the links

Comment (Tanya-Solyanik) [reply src/System.Drawing.Common/README.md:74]: And if you remove the query `?view=net-5.0`, the URL will automatically resolve to the newest shipped.

File: src/System.Drawing.Common/README.md [hunk 3]
```diff
@@ -12,5 +81,10 @@ Documentation can be found at https://learn.microsoft.com/en-us/dotnet/api/syste
 
 See the [Help Wanted](https://github.com/dotnet/runtime/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-System.Drawing+label%3A%22help+wanted%22) issues.
```
Comment (Tanya-Solyanik) [inline src/System.Drawing.Common/README.md:82]: Wrong link, should go to winforms now.

File: src/System.Drawing.Common/README.md [hunk 4]
```diff
@@ -12,5 +81,10 @@ Documentation can be found at https://learn.microsoft.com/en-us/dotnet/api/syste
...
 
-## Deployment
-`System.Drawing.Common` is provided as a [NuGet package](https://www.nuget.org/packages/System.Drawing.Common) and part of the `Microsoft.WindowsDesktop.App` shared framework.
```
Comment (Tanya-Solyanik) [inline src/System.Drawing.Common/README.md:16]: Do we have a good link for the WindowsDesktop package download?

File: src/System.Drawing.Common/README.md [hunk 5]
```diff
@@ -12,5 +81,10 @@ Documentation can be found at https://learn.microsoft.com/en-us/dotnet/api/syste
...
+## Feedback
+
+For feedback, you can:
```
Comment (Tanya-Solyanik) [inline src/System.Drawing.Common/README.md:86]: I would remove this line, you can change the title to `Feedback channels`  if you feel it needs clarifiction.

File: src/System.Drawing.Common/README.md [hunk 6]
```diff
@@ -12,5 +81,10 @@ Documentation can be found at https://learn.microsoft.com/en-us/dotnet/api/syste
...
+
+- Open an issue on the [GitHub repository](https://github.com/dotnet/runtime/issues)
+- Reach out on Twitter with the hashtag #dotnet
```
Comment (Tanya-Solyanik) [inline src/System.Drawing.Common/README.md:89]: Don't we have a winforms hashtag?

File: src/System.Drawing.Common/README.md [hunk 7]
```diff
@@ -1,9 +1,78 @@
 # System.Drawing.Common
-This assembly provides access to GDI+ basic graphics functionality via types such as [`Bitmap`](https://learn.microsoft.com/en-us/dotnet/api/system.drawing.bitmap) and [`Font`](https://learn.microsoft.com/en-us/dotnet/api/system.drawing.font).
 
-Note that `System.Drawing.Common` is only supported on Windows: https://learn.microsoft.com/en-us/dotnet/core/compatibility/core-libraries/6.0/system-drawing-common-windows-only.
+The System.Drawing.Common package allows .NET Core and .NET 5/6/7/8+ applications to access GDI+ graphics functionality. This package is especially useful for porting .NET Framework applications that rely on the `System.Drawing` namespace.
```
[the remark this answers is not in the document]
Comment (KlausLoeffelmann) [reply src/System.Drawing.Common/README.md:3]: There also was .NET Core 1.0, 2.0... While WinForms earliest was 3.1, I think System.Drawing used to be 1.0?

File: src/System.Drawing.Common/README.md [hunk 8]
```diff
@@ -1,9 +1,78 @@
...
 
-Documentation can be found at https://learn.microsoft.com/en-us/dotnet/api/system.drawing.
+## Getting Started
+
+To get started with System.Drawing.Common, install it using the NuGet Package Manager, the .NET CLI, or by editing your project file directly.
```
[the remark this answers is not in the document]
Comment (KlausLoeffelmann) [reply src/System.Drawing.Common/README.md:7]: > I think, it's not necessary to repeat `To get started...` , this is already clear from the section title. `Install System.Drawing.Common package using the NuGet Package Manager,`

I think it sounds nicer that way... 😉

File: src/System.Drawing.Common/README.md [hunk 9]
```diff
@@ -1,9 +1,78 @@
...
+
+### .NET CLI
+```
+dotnet add package System.Drawing.Common --version [your_version_here]
+```
+
+### Prerequisites
+
+- .NET Core 2.0-3.1 (min. .NET Core 3.1 for WinForms/WPF Apps)
+- .NET 5-.NET 8+
```
[the remark this answers is not in the document]
Comment (KlausLoeffelmann) [reply src/System.Drawing.Common/README.md:24]: Yeah, I think, let's agree to do it with `version+`.

File: src/System.Drawing.Common/README.md (hunk 4 above)
Comment (KlausLoeffelmann) [reply src/System.Drawing.Common/README.md:16]: Do you need that? I mean, the moment you use the `Sdk` attribute for the `Project` element in the csproj or the vbproj file and then specify `<UseWindowsForms>true</UseWindowsForms>`, the package gets added automatically, doesn't it?

I'll add this explanation to the file.

File: src/System.Drawing.Common/README.md [hunk 10]
```diff
@@ -1,16 +1,88 @@
...
+- [System.Drawing.Common Namespace | Microsoft Docs](https://docs.microsoft.com/dotnet/api/system.drawing)
+- [Drawing with System.Drawing.Common | Microsoft Learn](https://learn.microsoft.com/dotnet/core/drawing/)
 
 ## Contribution Bar
 - [x] [We consider new features, new APIs and performance changes](../README.md#primary-bar)
 - [x] [We consider PRs that target this library for new source code analyzers](../README.md#secondary-bars)
 - [ ] [We don't accept refactoring changes due to new language features](../README.md#secondary-bars)
 
-See the [Help Wanted](https://github.com/dotnet/runtime/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-System.Drawing+label%3A%22help+wanted%22) issues.
+See the [Help Wanted](https://github.com/dotnet/winforms/milestone/70) issues.
```
Comment (Tanya-Solyanik) [inline src/System.Drawing.Common/README.md:82]: This is a "no milestone" link, you probably want https://github.com/dotnet/winforms/issues?q=is%3Aopen+is%3Aissue+label%3Aarea-System.Drawing+label%3A%22help+wanted%22

File: src/System.Drawing.Common/README.md [hunk 11]
```diff
@@ -1,16 +1,88 @@
...
+## Additional Documentation
+
+For more in-depth tutorials and API references, you can check the following resources:
+
+- [NuGet Gallery | System.Drawing.Common](https://nuget.org/packages/System.Drawing.Common/)
+- [System.Drawing.Common Namespace | Microsoft Docs](https://docs.microsoft.com/dotnet/api/system.drawing)
+- [Drawing with System.Drawing.Common | Microsoft Learn](https://learn.microsoft.com/dotnet/core/drawing/)
 
 ## Contribution Bar
 - [x] [We consider new features, new APIs and performance changes](../README.md#primary-bar)
```
Comment (Tanya-Solyanik) [inline src/System.Drawing.Common/README.md:78]: I think these links to the upper level README are wrong

Comment (KlausLoeffelmann) [reply src/System.Drawing.Common/README.md:78]: Ah, those were links from the previous version.

File: src/System.Drawing.Common/README.md (hunk 4 above)
Comment (Tanya-Solyanik) [reply src/System.Drawing.Common/README.md:16]: I like the change you made, but this is complimentary to what I meant.  Something like this - https://github.com/dotnet/installer#installers-and-binaries

**Review by Tanya-Solyanik — APPROVED** [review]
I added a comment about a download link to .NET installers, but I don't feel strongly about it, take it or leave it.

**Pull request merged**